src/test/scala/performanceanalysis/logreceiver/alert/AlertActionActorSpec.scala

```diff
@@ -0,0 +1,55 @@
+package performanceanalysis.logreceiver.alert
+
+import akka.actor.{ActorSystem, Props}
+import akka.testkit.TestProbe
+import performanceanalysis.base.ActorSpecBase
+import performanceanalysis.server.Protocol._
+import akka.http.scaladsl.{Http, HttpExt}
+import akka.http.scaladsl.model.{HttpMethods, HttpRequest}
+import akka.stream.ActorMaterializer
+//import org.scalamock.scalatest.MockFactory
+import org.scalatest.mockito.MockitoSugar
+import org.mockito.Mockito.verify
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/scala-academy/performance-analysis/file/2712622796/issues/source?bid=3220915&fileBranchId=3305450#l12)